core/env/svid.go

```diff
@@ -38,21 +38,6 @@ func SafeSvidPrefix() string {
 	return p
 }
 
-// NotarySvidPrefix returns the prefix for the VSecM Notary SVID
```

Comment:
This is not being used, and will likely not be designed for a long time.

We can introduce it when the time comes.